cmd/juju/cloud/add.go

```diff
@@ -721,7 +721,7 @@ func (p *CloudFileReader) ReadCloudFromFile(cloudFile string, ctxt *cmd.Context)
 		return nil, errors.Trace(err)
 	}
 	if len(specifiedClouds) == 0 {
-		return nil, errors.New("no personal clouds are defined")
+		return nil, errors.New("no clouds found in parsed yaml, please validate yaml keys")
```

Comment:
Is the error message that we get in the two sets of lines above also going to be vague? I think it might make sense to wrap all three errors in something like: ```"Error parsing yaml: %s" % err```  (That's pseudo code Python style formatting -- obv. you'd want to do the Golang equivalent :-) )